test/backend/remote.js

```diff
@@ -44,7 +44,9 @@ function respOkEmpty(t, r) {
 // Takes the test object (t) and the error object (err).
 function testNotFound(t, err) {
   t.equal(err.statusCode, 404, 'returns a 404')
-  t.deepEqual(err.body, { message : 'Not Found' }, 'Object contains no other fields')
+  t.deepEqual(err.body, {
+    message : 'Not Found', errno: 116, error: 'Not Found', code: 404
```

Comment:
nit: maybe put each of these props on a separate line for readability.
Also, remove the stray space between `message` and `:` since our linter may blow up (but not sure in this exact repo).